frontend/benefit/applicant/src/utils/common.ts

```diff
@@ -16,3 +16,14 @@ export const getLanguageOptions = (
 
   return createOptions(Object.values(SUPPORTED_LANGUAGES));
 };
+
+export const getApplicationStepFromString = (step: string): number => {
+  try {
+    return parseInt(step.split('_')[1], 10);
```

Comment:
Why radix parameter `10` is needed?

Reply:
because linter complains if you dont define any

Reply:
default for decimal si 10, so i think its ok